internal/app/rds/rds.go

```diff
@@ -272,6 +275,8 @@ func (r *RDSFetcher) computeInstanceMetrics(dbInstance aws_rds_types.DBInstance,
 		Status:                     GetDBInstanceStatusCode(*dbInstance.DBInstanceStatus),
 		StorageThroughput:          converter.MegaBytesToBytes(storageThroughput),
 		StorageType:                *dbInstance.StorageType,
+		CACertificateIdentifier:    *dbInstance.CACertificateIdentifier,
+		CACertificateValidTill:     *dbInstance.CertificateDetails.ValidTill,
```

Comment:
`CACertificateValidTill` suggests that it's the CA certificate expiration, but here it's the server certificate expiration date.

```suggestion
		CertificateValidTill:     *dbInstance.CertificateDetails.ValidTill,
```

Reply:
Good point, I've missed that!

For my own information, this value is expected to be under 6 months, as the new one will be changed at half life:

> RDS attempts to rotate your DB server certificate in your preferred maintenance window at the **DB server certificate half life**. The new DB server certificate is valid for 12 months.
[AWS doc](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL-certificate-rotation.html#UsingWithRDS.SSL-certificate-rotation-server-cert-rotation)

Except for Instances that do not support Automatic certificate rotation (Only SQL Server).